src/assets/release-notes-1.28.0-alpha.3.json

```diff
@@ -0,0 +1,364 @@
+    "areas": ["kube-proxy"],
+    "kinds": ["cleanup"],
+    "sigs": ["network"]
+  },
+  "118128": {
+    "commit": "173a4738035b186f50c5882546155b13bb7f576d",
+    "text": "kube-controller-manager deprecate `--volume-host-cidr-denylist` and `--volume-host-allow-local-loopback` flags.",
+    "markdown": "Kube-controller-manager deprecate `--volume-host-cidr-denylist` and `--volume-host-allow-local-loopback` flags. ([#118128](https://github.com/kubernetes/kubernetes/pull/118128), [@carlory](https://github.com/carlory)) [SIG API Machinery, Apps, Network, Node, Storage and Testing]",
+    "author": "carlory",
+    "author_url": "https://github.com/carlory",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/118128",
+    "pr_number": 118128,
+    "areas": ["test", "kubelet", "kube-proxy", "code-generation"],
+    "kinds": ["cleanup", "api-change", "deprecation"],
+    "sigs": ["network", "storage", "node", "api-machinery", "apps", "testing"],
+    "duplicate": true,
+    "duplicate_kind": true
+  },
+  "118278": {
+    "commit": "1de217b095988a0e76b2ac4f3ebcfcbf4bb5f08e",
+    "text": "Update the comment about the feature-gate level for PodFailurePolicy from alpha to beta",
+    "markdown": "Update the comment about the feature-gate level for PodFailurePolicy from alpha to beta ([#118278](https://github.com/kubernetes/kubernetes/pull/118278), [@mimowo](https://github.com/mimowo)) [SIG Apps]",
+    "author": "mimowo",
+    "author_url": "https://github.com/mimowo",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/118278",
+    "pr_number": 118278,
+    "kinds": ["bug", "documentation", "api-change"],
+    "sigs": ["apps"],
+    "duplicate_kind": true
+  },
+  "118329": {
+    "commit": "9d231f8ce77b4a8413b41130ea130d392ca860c7",
+    "text": "Fixes the alpha `CloudDualStackNodeIPs` feature.",
+    "markdown": "Fixes the alpha `CloudDualStackNodeIPs` feature. ([#118329](https://github.com/kubernetes/kubernetes/pull/118329), [@danwinship](https://github.com/danwinship)) [SIG Network and Node]",
+    "documentation": [
+      {
+        "description": "[KEP]",
+        "url": "https://github.com/kubernetes/enhancements/issues/3705",
+        "type": "KEP"
+      }
+    ],
+    "author": "danwinship",
+    "author_url": "https://github.com/danwinship",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/118329",
+    "pr_number": 118329,
+    "areas": ["kubelet"],
+    "kinds": ["bug", "feature"],
+    "sigs": ["network", "node"],
+    "feature": true,
+    "duplicate": true,
+    "duplicate_kind": true
+  },
+  "118420": {
+    "commit": "b63700630231803967f85e83ff35d719db511baa",
+    "text": "ACTION_REQUIRED\nWhen an Indexed Job has a number of completions higher than 10^5 and parallelism higher than 10^4, and a big number of Indexes fail, Kubernetes might not be able to track the termination of the Job. Kubernetes now emits a warning, at Job creation, when the Job manifest exceeds both of these limits.",
```

Comment:
I think we can remove the ACTION_REQUIRED, as they are not a part of the release note.